examples/PowerElectronics/Microgrid/Microgrid.cpp

```diff
@@ -421,11 +424,31 @@ int main(int argc, char const* argv[])
       3.604108939430972e+02,
       -3.492842627398574e+01};
 
-  std::cout << "Test the Relative Error\n";
+  // std::cout << "Test the Relative Error\n";
```

Comment:
Consider adding a verbosity flag and a conditional to get to this code. Or remove completely.

Reply:
This is reminder for us to add verbosity flags. We'll do it once CLI utilities are added to GridKit. For now, it is out of scope.